index.fr.md

```diff
@@ -0,0 +1,147 @@
+    Pour plus d’informations, voir <a href="https://www.w3.org/WAI/about/usingWAImaterial.html" lang="en" hreflang="en">Using WAI Materials (en anglais)</a>.
+  </p>
+  <p><strong>Date&nbsp;: </strong>Mis à jour le 27 février 2018. <!-- [<a href="@@">Changelog</a>] --> </p>
+  <p><strong>Éditrice&nbsp;:</strong> <a href="http://www.w3.org/People/Shawn" lang="en" hreflang="en">Shawn Lawton Henry</a>. Graphiste&nbsp;: Michael Duffy.</p>
+  <p>Élaboré avec des contributions du groupe de travail «&nbsp;Éducation et Sensibilisation&nbsp;» (<span lang=”en”>Education and Outreach Working Group</span>) (<a href="http://www.w3.org/WAI/EO/">EOWG</a>).</p>
+ 
+ 
+# Read Important Translations Guidance at https://www.w3.org/WAI/about/translating/#important
+# Read Translations Notes for this resource at https://github.com/w3c/wai-components/blob/master/README.md
+# end of translation instructions 
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Résumé" class="full" %}
+{:/}
+ 
+Cette page montre comment l’accessibilité dépend de l’articulation harmonieuse de nombreuses composantes, et comment des améliorations de certaines pourraient améliorer sensiblement l’accessibilité du web.
+ 
+Elle fournit les bases pour comprendre les différents standards d’accessibilité produits par l’Initiative pour l’Accessibilité du Web (<span lang="en">Web Accessibility Initiative WAI</span>) du <abbr title="World Wide Web Consortium">W3C</abbr>.
+ 
+{::nomarkdown}
+{% include box.html type="end" %}
+{:/}
+ 
+{::nomarkdown}
+{% include_cached toc.html type="start" title="Contenu de la page" class="simple" %}
+{:/}
+ 
+{::options toc_levels="2" /}
+ 
+-   This text will be replaced by the TOC.
+{:toc}
+ 
+{::nomarkdown}
+{% include_cached toc.html type="end" %}
+{:/}
+ 
+ 
+## Introduction {#intro}
+{:.no_toc}
+ 
+Il est essentiel que des composantes distinctes et multiples du développement web et de l’interaction utilisateur s’articulent convenablement pour que le Web soit accessible aux personnes handicapées. Ces composantes incluent&nbsp;:
+ 
+-   **du contenu** – l’information contenue dans une page web ou une application web, y compris&nbsp;:
+	-   de l’information primaire telle que du texte, des images et des sons,
+	-   du code ou du balisage qui définit la structure, la présentation… &nbsp;;
+-   **des navigateurs web, des lecteurs de média**, et d’autres «&nbsp;agents utilisateurs&nbsp;»&nbsp;;
+-   **une technologie d’assistance**, dans certains cas – lecteurs d’écran, claviers alternatifs, contacteurs, logiciel de numérisation… &nbsp;;
+-   les connaissances, l’expérience et dans certains cas les stratégies d’adaptation **des utilisateurs** pour l’utilisation du Web&nbsp;;
+-   **des créateurs** – concepteurs, graphistes, codeurs, auteurs… y compris des créateurs handicapés et des utilisateurs qui contribuent&nbsp;;
```

Comment:
Context: https://github.com/w3c/translation-glossaries/blob/main/fran%C3%A7ais-French.md#people-with-disabilities

```suggestion
-   **des créateurs** – concepteurs, graphistes, codeurs, auteurs, etc., y compris des créateurs en situation de handicap et des utilisateurs qui contribuent au contenu&nbsp;;
```

Reply:
I disagree. The French translation of the UN Convention is « Convention relative aux droits des **personnes handicapées** » cf
https://www.ohchr.org/fr/instruments-mechanisms/instruments/convention-rights-persons-disabilities
See https://www.service-public.fr/particuliers/vosdroits/F12242
You can be in a « situation de handicap » without any disability. It’s a expression from the domain of Human factors and ergonomics (ergonomie).
And see https://www.w3.org/Translations/WCAG21-fr/#dfn-ambiguous-to-users-in-general

Reply:
@3l3gant-cod3s Very interesting perspective!

In your view, this would mean using "personnes handicapées" or "personnes en situation de handicap" depending on context?

I would like to get input from other translators on that. I have opened an issue: https://github.com/w3c/translation-glossaries/issues/32

Reply:
ping @3l3gant-cod3s